tests/Microsoft.Fx.Portability.Tests/BreakingChangeTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace Microsoft.Fx.Portability.Tests
+{
+    public class BreakingChangeTests
+    {
+        [Fact]
+        public void EqualityTests()
+        {
+            var breakingChangeSame1 = new BreakingChange { Id = "id1" };
+            var breakingChangeSame2 = new BreakingChange { Id = "id1" };
+            var breakingChangeDifferent1 = new BreakingChange { Id = "ID1" };
+            var breakingChangeDifferent2 = new BreakingChange { Id = "id2" };
```

Comment:
Since the equality is case sensitive (which I think is probably correct), we should maybe add a test case to confirm that id1 and ID1 are not equal (just to make sure that behavior isn't inadvertently changed in the future).